systemtest/src/main/java/io/strimzi/systemtest/resources/operator/SetupClusterOperator.java

```diff
@@ -13,12 +13,26 @@
 
 public class SetupClusterOperator {
 
+    private static SetupClusterOperator instance;
     private ClusterOperatorConfiguration clusterOperatorConfiguration = new ClusterOperatorConfiguration();
 
     private HelmInstallation helmInstallation;
     private OlmInstallation olmInstallation;
     private YamlInstallation yamlInstallation;
 
+    // Private constructor to prevent external instantiation
+    private SetupClusterOperator() {
+        this.clusterOperatorConfiguration = new ClusterOperatorConfiguration();
+    }
+
+    // Public method to get the singleton instance
+    public static synchronized SetupClusterOperator get() {
+        if (instance == null) {
+            instance = new SetupClusterOperator();
+        }
+        return instance;
+    }
```

Comment:
I think you can make it via lazy evaluation instead of this. Simply just
```java
public class SetupClusterOperator {
     private SetupClusterOperator() {}
...

    private static class Holder {
           private static final SetupClusterOperator INSTANCE = new SetupClusterOperator();
    }
    
    public static SetupClusterOperator get() {  // here I would rather name is getInstance() instead of get() as that's convention for most of the Singletons.
        return Holder.INSTANCE;
    }
}
```
Also, JVM guarantees that class initialization is thread-safe, so there is no need for the synchronised keyword within `get()`.

Reply:
Great, thanks :) looks better than how I have it now.